examples/nirf/specan-nr-txp-example.py

```diff
@@ -0,0 +1,343 @@
+            instrument=instr,
+            selector_string="",
+            attribute_id=nirfmxspecan_types.NIRFMXSPECAN_ATTRIBUTE_SELECTED_PORTS,
+            attr_val_raw="",
+        )
+    )
+
+    specanclient.CfgRF(
+        nirfmxspecan_types.CfgRFRequest(
+            instrument=instr,
+            selector_string="",
+            center_frequency=1e9,
+            reference_level=0.00,
+            external_attenuation=0.00,
+        )
+    )
+
+    specanclient.TXPCfgMeasurementInterval(
+        nirfmxspecan_types.TXPCfgMeasurementIntervalRequest(
+            instrument=instr,
+            selector_string="",
+            measurement_interval=1e-3,
+        )
+    )
+
+    specanclient.TXPCfgRBWFilter(
+        nirfmxspecan_types.TXPCfgRBWFilterRequest(
+            instrument=instr,
+            selector_string="",
+            rbw=100e3,
+            rbw_filter_type=nirfmxspecan_types.TXP_RBW_FILTER_TYPE_GAUSSIAN,
+            rrc_alpha=0.1,
+        )
+    )
+
+    specanclient.TXPCfgAveraging(
+        nirfmxspecan_types.TXPCfgAveragingRequest(
+            instrument=instr,
+            selector_string="",
+            averaging_enabled=nirfmxspecan_types.TXP_AVERAGING_ENABLED_FALSE,
+            averaging_count=10,
+            averaging_type=nirfmxspecan_types.TXP_AVERAGING_TYPE_RMS,
+        )
+    )
+
+    ### Fetch SpecAn measurement data ###
+
+    txp_read_response = specanclient.TXPRead(
+        nirfmxspecan_types.TXPReadRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
+        )
+    )
+    check_for_warning(txp_read_response, instr)
+
+    minimum_power = txp_read_response.minimum_power
+    maximum_power = txp_read_response.maximum_power
+    peak_to_average_ratio = txp_read_response.peak_to_average_ratio
+    average_mean_power = txp_read_response.average_mean_power
+
+    print("\n------------SpecAn TXP Measurement------------\n")
+    print(f"Average Mean Power(dBm)    {average_mean_power}")
+    print(f"Peak to Average Ratio(dB)  {peak_to_average_ratio}")
+    print(f"Maximum Power(dBm)         {maximum_power}")
+    print(f"Minimum Power(dBm)         {minimum_power}")
+except grpc.RpcError as rpc_error:
+    error_message = rpc_error.details()
+    for entry in rpc_error.trailing_metadata() or []:
+        if entry.key == "ni-error":
+            value = entry.value if isinstance(entry.value, str) else entry.value.decode("utf-8")
+            error_message += f"\nError status: {value}"
+    if rpc_error.code() == grpc.StatusCode.UNAVAILABLE:
+        error_message = f"Failed to connect to server on {SERVER_ADDRESS}:{SERVER_PORT}"
+    elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
+        error_message = (
+            "The operation is not implemented or is not supported/enabled in this service"
+        )
+    sys.stderr.write(f"{error_message}\n")
+finally:
+    if instr:
+        nrclient.Close(

```

Comment:
```suggestion
        instrclient.Close(
```
This should just be on the `instrclient` and not the NR one.